platform/broadcom/sonic-platform-modules-dell/z9664f/sonic_platform/fan.py

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env python
+
+########################################################################
+# DELL Z9664F
+#
+# Module contains an implementation of SONiC Platform Base API and
+# provides the Fans' information which are available in the platform.
+#
+########################################################################
+try:
+    from sonic_platform_base.fan_base import FanBase
+    from sonic_platform.ipmihelper import IpmiSensor, IpmiFru
+except ImportError as e:
+    raise ImportError(str(e) + "- required module not found")
+
+FAN1_MAX_SPEED_OFFSET = 71
```

Comment:
FAN1_MAX_SPEED_OFFSET = 71
FAN2_MAX_SPEED_OFFSET = 73
PSU_FAN_MAX_SPEED_OFFSET = 50 . Should these values be changed for this platform ?

Reply:
It should be same for the BMC platforms.